documentation/modules/con-standard-subscription.adoc

```diff
@@ -6,13 +6,9 @@
 = Subscription
 // !standard.address.subscription.shortDescription:A subscription on a specified topic
 // !standard.address.subscription.longDescription:start
-The subscription address type allows a subscription to be created for a topic that holds messages
-published to the topic even if the subscriber is not attached. The subscription is accessed by the
-consumer using <topic-address>::<subscription-address>. For example, for a subscription `mysub` on a
-topic `mytopic` the consumer consumes from the address `mytopic::mysub`. By default, only a single
-consumer is allowed per subscription. This can be specified using `maxConsumers` setting of
-the subscription address.
+Using the subscription address type you can create a subscription for a topic that holds messages
+published to the topic even if the subscriber is not attached. The consumer accesses the subscription  using the following address syntax: <topic-address>::<subscription-address>. For example, for a subscription `mysub` on a topic `mytopic` the consumer accesses the subscription from the address `mytopic::mysub`. The default setting permits only a single consumer per subscription. This setting can be changed by editing the `maxConsumers` attribute of the subscription address.
```

Comment:
we say field rather than attribute.
```suggestion
published to the topic even if the subscriber is not attached. The consumer accesses the subscription  using the following address syntax: <topic-address>::<subscription-address>. For example, for a subscription `mysub` on a topic `mytopic` the consumer accesses the subscription from the address `mytopic::mysub`. The default setting permits only a single consumer per subscription. This setting can be changed by editing the `maxConsumers` field of the subscription address.
```

Reply:
@k-wall apologies, I missed this comment before I clicked Merge. Opening an issue and assigning to Heather M. to fix